tests/test_api.py

```diff
@@ -1,14 +1,17 @@
 import os         # used to get the location of the testdata
 import sys
 import geoextent
+sys.path.append("../geoextent/")
```

Comment:
Use relative imports, see https://stackoverflow.com/a/4542391

`from ..geoextent import extractFromFolderOrFile`

Reply:
using relative imports made it work on tests properly but created error when running the library. Error message is ("attempted relative import with no known parent package")

https://stackoverflow.com/questions/55041141/importerror-attempted-relative-import-with-no-known-parent-package